synapse/handlers/auth.py

```diff
@@ -132,14 +131,14 @@ def __init__(self, hs):
         # after the SSO completes and before redirecting them back to their client.
         # It notifies the user they are about to give access to their matrix account
         # to the client.
-        self._sso_redirect_confirm_template = load_jinja2_templates(
-            hs.config.sso_template_dir, ["sso_redirect_confirm.html"],
+        self._sso_redirect_confirm_template = hs.config.read_templates(
+            ["sso_redirect_confirm.html"], hs.config.sso_template_dir
         )[0]
```

Comment:
We seem to inconsistently read templates in the config vs. in the handlers. I'm not sure if there's a benefit to making it more consistent, just thought I would mention it!

Reply:
Are you just referring to the different `filters` value? Or the `(bla,) = ...` vs `...[0]`?

Reply:
I'm referring to this code being in `synapse.handlers` and not `synapse.config`.

Reply:
Although I did notice the `(bla,) = ...` vs `...[0]`, but didn't think it was worth worrying about. :)

Reply:
Added a commit to clean all this up :)